e2e/walletAvatarOptions.spec.js

```diff
@@ -8,95 +9,123 @@ const DISMISS_AVATAR_BUILDER_COORDS = { x: 20, y: 90 };
+      await Helpers.checkIfExistsByText('Pick an Emoji');
+    }
     await Helpers.checkIfExistsByText('Create your Profile');
-    await Helpers.tapByText('Pick an Emoji');
-    await Helpers.tapAtPoint('avatar-builder', DISMISS_AVATAR_BUILDER_COORDS);
-    await Helpers.tapAtPoint('profile-screen', WALLET_AVATAR_COORDS);
     await Helpers.tapByText('Create your Profile');
     await Helpers.checkIfVisible('ens-intro-sheet');
     await Helpers.swipe('ens-intro-sheet', 'down', 'slow');
   });
 
-  it('watch wallet with ens but without ens avatar', async () => {
+  it('watch wallet with ENS but without ENS avatar', async () => {
     await Helpers.tapAtPoint('profile-screen', WALLET_ADDRESS_COORDS);
     await Helpers.tapByText('􀂍 Add an existing wallet');
     await Helpers.typeText('import-sheet-input', RAINBOW_TEST_WALLET, false);
     await Helpers.waitAndTap('import-sheet-button');
     await Helpers.waitAndTap('wallet-info-submit-button');
-    await Helpers.relaunchApp();
-    await Helpers.swipe('wallet-screen', 'right', 'slow');
-    await Helpers.checkIfVisible('profile-screen');
   });
 
-  it('test watched wallet with ens but without ens avatar', async () => {
+  it('test watched wallet with ENS but without ENS avatar', async () => {
     await Helpers.tapAtPoint('profile-screen', WALLET_AVATAR_COORDS);
     await Helpers.checkIfExistsByText('Choose from Library');
-    await Helpers.checkIfExistsByText('Pick an Emoji');
+    if (android) {
+      await Helpers.checkIfExistsByText('Shuffle Emoji');
+    } else {
+      await Helpers.checkIfExistsByText('Pick an Emoji');
+    }
     await Helpers.checkIfExistsByText('View Profile');
-    await Helpers.tapByText('Pick an Emoji');
-    await Helpers.checkIfVisible('avatar-builder');
-    await Helpers.tapAtPoint('avatar-builder', DISMISS_AVATAR_BUILDER_COORDS);
-    await Helpers.tapAtPoint('profile-screen', WALLET_AVATAR_COORDS);
+    if (!android) {
+      await Helpers.tapByText('Pick an Emoji');
+      await Helpers.checkIfVisible('avatar-builder');
+      await Helpers.tapAtPoint('avatar-builder', DISMISS_AVATAR_BUILDER_COORDS);
+      await Helpers.tapAtPoint('profile-screen', WALLET_AVATAR_COORDS);
+    }
     await Helpers.tapByText('View Profile');
     await Helpers.checkIfVisible('profile-sheet');
     await Helpers.swipe('profile-sheet', 'down', 'slow');
   });
 
-  it('import wallet with ens but without ens avatar', async () => {
+  it('import wallet with ENS but without ENS avatar', async () => {
     await Helpers.tapAtPoint('profile-screen', WALLET_ADDRESS_COORDS);
     await Helpers.tapByText('􀂍 Add an existing wallet');
     await Helpers.typeText('import-sheet-input', process.env.TEST_SEEDS, false);
     await Helpers.waitAndTap('import-sheet-button');
     await Helpers.waitAndTap('wallet-info-submit-button');
-    await Helpers.relaunchApp();
-    await Helpers.swipe('wallet-screen', 'right', 'slow');
-    await Helpers.checkIfVisible('profile-screen');
```

Comment:
I removed all relaunches from this file because they were effectively checking if profile screen still exists.

We can (and should) add proper checks if the wallet address matches etc. but in current form it was useless.